src/mme-app/actionHandlers/erabModIndicationActionHandlers.cpp

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2020-present Infosys Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/******************************************************************************
+ *
+ * This file has both generated and manual code.
```

Comment:
Is this true ? manual code as well as generated code ??

Reply:
yes.

Reply:
Let's discuss in meeting.